### ISSUE TITLE
BangFormat raised unexpected error linting file

### ISSUE THREAD

**Issue by pavelbinar** [brigade/scss-lint#316]
Error reported by WebStorm 9.0.2, Mac OS X 10.10.1.

```
SCSS Lint Plugin
SCSSLint::Linter::BangFormat raised unexpected error linting file client/css/_next-actions_scsslint_tmp3712180039925449666.scss: '[BUG] Unknown operator nil' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/parser.rb:206:in `precedence_of' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/tree/list_literal.rb:33:in `block in to_sass' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/tree/list_literal.rb:32:in `map' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/tree/list_literal.rb:32:in `to_sass' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/prop_node.rb:132:in `val_to_sass' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/prop_node.rb:109:in `declaration' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/convert.rb:233:in `visit_prop' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/convert.rb:10:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/node.rb:201:in `to_sass' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter/bang_format.rb:7:in `visit_prop' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:134:in `block in visit_children' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `each' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `visit_children' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:134:in `block in visit_children' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `each' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `visit_children' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:134:in `block in visit_children' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `each' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `visit_children' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:20:in `run' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:59:in `run_linter' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:42:in `block in find_lints' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:37:in `each' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:37:in `find_lints' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:21:in `block in run' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:20:in `each' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:20:in `run' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/cli.rb:115:in `run' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/bin/scss-lint:8:in `block in  ' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/bin/scss-lint:6:in `tap' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/bin/scss-lint:6:in ` ' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/scss-lint:23:in `load' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/scss-lint:23:in ` ' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval' /Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in ` ' Report this bug at https://github.com/causes/scss-lint/issues
```

```
12:10:04 PM SCSS Lint Plugin: SCSSLint::Linter::BangFormat raised unexpected error linting file client/css/_next-actions_scsslint_tmp3712180039925449666.scss: '[BUG] Unknown operator nil'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/parser.rb:206:in `precedence_of'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/tree/list_literal.rb:33:in `block in to_sass'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/tree/list_literal.rb:32:in `map'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/script/tree/list_literal.rb:32:in `to_sass'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/prop_node.rb:132:in `val_to_sass'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/prop_node.rb:109:in `declaration'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/convert.rb:233:in `visit_prop'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/convert.rb:10:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/node.rb:201:in `to_sass'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter/bang_format.rb:7:in `visit_prop'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:134:in `block in visit_children'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `each'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `visit_children'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:134:in `block in visit_children'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `each'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `visit_children'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:134:in `block in visit_children'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `each'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:132:in `visit_children'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:124:in `visit'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/linter.rb:20:in `run'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:59:in `run_linter'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:42:in `block in find_lints'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:37:in `each'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:37:in `find_lints'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:21:in `block in run'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:20:in `each'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/runner.rb:20:in `run'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/lib/scss_lint/cli.rb:115:in `run'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/bin/scss-lint:8:in `block in '
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/bin/scss-lint:6:in `tap'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/gems/scss-lint-0.30.0/bin/scss-lint:6:in `'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/scss-lint:23:in `load'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/scss-lint:23:in `'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `eval'
/Users/pavelbinar/.rvm/gems/ruby-2.1.2/bin/ruby_executable_hooks:15:in `'
Report this bug at https://github.com/causes/scss-lint/issues
```
